spec/document/documents_spec.rb

```diff
@@ -16,6 +16,73 @@ def read_file(dir_path, file_name)
     File.read(File.join(dir_path, file_name)).strip
   end
 
+  context 'A FinancialDocumentV1' do
+    context 'when processing an invoice' do
+      it 'should load an empty document prediction' do
+        response = load_json(DIR_FINANCIAL_DOCUMENT_V1, 'empty.json')
+        document = Mindee::Document.new(Mindee::Prediction::FinancialDocumentV1, response['document'])
+        expect(document.inference.product.type).to eq('standard')
+        prediction = document.inference.prediction
+        expect(prediction.invoice_number.value).to be_nil
+        expect(prediction.invoice_number.polygon).to be_empty
+        expect(prediction.invoice_number.bounding_box).to be_nil
+        expect(prediction.date.value).to be_nil
+        expect(prediction.date.page_id).to be_nil
+      end
+
+      it 'should load a complete document prediction' do
+        to_string = read_file(DIR_FINANCIAL_DOCUMENT_V1, 'summary_full_invoice.rst')
+        response = load_json(DIR_FINANCIAL_DOCUMENT_V1, 'complete_invoice.json')
+        document = Mindee::Document.new(Mindee::Prediction::FinancialDocumentV1, response['document'])
+        prediction = document.inference.prediction
+        expect(prediction.invoice_number.bounding_box.top_left.x).to eq(prediction.invoice_number.polygon[0][0])
+        expect(prediction.date.value).to eq('2019-02-11')
+        expect(prediction.due_date.value).to eq('2019-02-26')
+        expect(prediction.due_date.page_id).to eq(0)
+        expect(document.to_s).to eq(to_string)
+      end
+
+      it 'should load a complete page 0 prediction' do
+        to_string = read_file(DIR_FINANCIAL_DOCUMENT_V1, 'summary_page0_invoice.rst')
+        response = load_json(DIR_FINANCIAL_DOCUMENT_V1, 'complete_invoice.json')
+        document = Mindee::Document.new(Mindee::Prediction::FinancialDocumentV1, response['document'])
+        page = document.inference.pages[0]
+        expect(page.orientation.value).to eq(0)
+        expect(page.prediction.due_date.page_id).to eq(0)
+        expect(page.to_s).to eq(to_string)
```

Comment:
This fails because `[summary_full_receipt.rst](https://github.com/mindee/client-lib-test-data/compare/643f4331b6ea799509952079e8ba504df21fd4a8...c969b664cc7ce41aae651f055ddb009a93dc722b#diff-9d8284c1a60e3c114bde590ac42c2efc833385d13990db3a5c9f4a3642d362d3)` seems to have an extra blank line at 37-38

Reply:
Pushed a fix to the data files.